pxtblocks/plugins/comments/blockComment.ts

```diff
@@ -246,6 +266,15 @@ export class CommentIcon extends Blockly.icons.Icon {
         }
     }
 
+    onPositionChange(): void {
+        if (this.textInputBubble) {
+            const coord = this.textInputBubble.getPositionRelativeToAnchor();
+
+            setBlockDataForField(this.sourceBlock, COMMENT_OFFSET_X_FIELD_NAME, coord.x + "");
```

Comment:
Why is the `+ ""` needed for the coordinate?

Reply:
it's a number, needs to be a string

Reply:
Woah, I didn't know that was a thing, thanks for the info!